docs/_news/news06.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: news
+title: New for Codewind 0.6.0
+description: New for Codewind 0.6.0
+keywords: news, new, updates, update, version
+duration: 1 minute
+permalink: news06
+---
+
+## Codewind 0.6.0
+Thursday 21 November 2019
+
+#### Codewind workspace updates
+**VS Code**
+- You no longer need to place projects in the `codewind-workspace`.
+- Do not create projects in the `~/codewind-data/` or `C:\codewind-data` directories. User configuration files are now kept in `~/.codewind/` or `C:\Users\<user>\.codewind\`.
+- If you want to move an existing project, remove it from Codewind without deleting the files. Next, move the project to its new location. Then, run the `Add Existing Project` command to add it to its new location.
+- In VS Code, to skip the **Select a parent directory** step when you create a project, use the `codewind.alwaysCreateProjectsInWorkspace` setting.
+- Project information is now kept in the `codewind_cw-workspace` Docker volume. Information for existing projects is moved to the Docker volume when Codewind 0.6.0 is started for the first time. If this volume is deleted, your projects are deleted from Codewind. However, they still exist on your disk, and you can add them back into Codewind with the `Add Existing Project` command.
+- Whenever files in the project are changed, the changed files are copied into the `codewind_cw-workspace` Docker volume. Previously, Codewind did a Docker mount of the entire `codewind-workspace`.
+**Eclipse**
+The information in the **VS Code** section also applies to Eclipse but with these exceptions:
+- When you move a project, to add an existing project to Eclipse, right-click **Local** in the Codewind explorer view and select **Add Existing Project**.
+- Select the location of the project when you create it. The Eclipse workspace is the default location, but you can clear the **Use default location** checkbox and enter another location in the **Location** field.
+
+#### Appsody
+- Appsody works on Che.
```

Comment:
this is a pretty big deal, can we at least add an exclamation? 😅

Reply:
@tetchel Despite an award-winning author convincing me that less is more when it comes to exclamation points, I added one for you, Tim. 😉 

If we want some more text to elaborate on this point, though, please let me know how we should highlight it. 👍